src/DataFactory/DataFactoryV2/DataFactories/SetAzureDataFactoryCommand.cs

```diff
@@ -99,6 +99,16 @@ public class SetAzureDataFactoryCommand : DataFactoryBaseCmdlet
         #endregion // Attributes
         public string Name { get; set; }
 
+        #region Attributes
+        [Parameter(Mandatory = false, HelpMessage = Constants.HelpIdentityType)]
+        #endregion
+        public string IdentityType { get; set; }
+
+        #region Attributes
+        [Parameter(Mandatory = false, HelpMessage = Constants.HelpUserAssignedIdentities)]
+        #endregion
+        public IDictionary<string,object> UserAssignedIdentities { get; set; }
```

Comment:
```suggestion
        public IDictionary<string,object> UserAssignedIdentity { get; set; }
```
We should keep the singularity of parameter. Please refer to: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/parameter-best-practices.md#singularity